.github/CODEOWNERS

```diff
@@ -1,13 +1,5 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
-# Primary repo maintainers
```

Comment:
Any particular reason for these changes?

Reply:
[Changes committed](https://github.com/cosmos/cosmos-sdk/pull/7137) as per Launchpad's release policy. The Policy states that PRs to the stable release branch must be approved by the Stable Release Managers.
Changes were approved offline by @okwme.

Reply:
Ok this is just for launchpad? 

Also, why is the base branch `launchpad/release/v0.39.2`? It should be `release/v0.39.x` or something similar.

Reply:
> Ok this is just for launchpad?

Currently yes, correct - although IIRC people liked the concept of having a small dedicated group of people maintaining each stable release branch. Yet yes of course, this applies to Launchpad only.

> Also, why is the base branch launchpad/release/v0.39.2?

There should be ACLs enforced on branches that start with `launchpad/`

Reply:
Anyways, the release branch will be removed once the final tag is pushed

Reply:
I'm not sure I understand but ok!